specifications/M1/dsp.profile.md

```diff
@@ -0,0 +1,49 @@
+# 1. Introduction
+
+This document defines a profile of
+the [Dataspace Protocol Specifications (DSP)](https://github.com/International-Data-Spaces-Association/ids-specification).
+
+# Catalog Metadata for the __Well-Known__ Endpoint
+
+Catalogs that support the [Credential Issuance Protocol (CIP)](credential.issuance.protocol.md) MUST return a JSON
+object from the `/.well-known/dspace-trust` endpoint with the following properties:
+
+- `@context`: REQUIRED. Specifies a valid [Json-Ld context](https://www.w3.org/TR/json-ld11/#the-context).
+- `credentialsSupported`: REQUIRED. Contains a Json structure corresponding to the schema
+  specified [below](#the-credentialssupported-object).
+
+A non-normative example is:
+
+```json
+{
+  "@context": [
+    ...
+  ],
+  "credentialsSupported": [
+    {
+      "types": [
+        "VerifiableCredential",
+        "CompanyCredential"
+      ],
+      "bindingMethod": "did:web",
+      "cryptographicSuite": "JsonWebKey2020"
```

Comment:
technically, the suite is called [JsonWebSignature2020](https://www.w3.org/community/reports/credentials/CG-FINAL-lds-jws2020-20220721/). JsonWebKey is merely how keys are represented.

```suggestion
      "cryptographicSuite": "JsonWebSignature2020"
```

Reply:
I think that is what other specs are using. In any event, this section is non-normative and we can raise a separate issue as to what we want to do.